brave/brave-ios
# Fix #195  Added Migration code for sqlite files from Document to Application Support folder

Sqlite file is now migrated to the new folder. There two questions that need to be answered/invetigated before merge.
1. How will this function when there is data model version change(ie DB migration).
2. Before we allow iTunesFileSharing, we need to make sure all contents of Documents Directory are allowed for user visibility.
Fore Example: `https-everywhere-data` folder and Locally saved cookies(can be moved to Application support).



## Submitter Checklist:



## Test Plan:
We have to test if the app upgrades correctly so:
- install previous app version, add few bookmarks, visit some websites so history is filled
- upgrade to version with this PR merged
- verify it doesn't crash, and Bookmarks/History are staying



### Screenshots:
Before:
<img width="967" alt="Screenshot 2019-07-25 at 12 22 59 PM" src="https://user-images.githubusercontent.com/5398557/61853117-bdae5a80-aed8-11e9-9ab5-c822587f76b4.png">
After:
<img width="943" alt="Screenshot 2019-07-25 at 12 25 03 PM" src="https://user-images.githubusercontent.com/5398557/61853127-c0a94b00-aed8-11e9-97ba-d58867ba4ca0.png">





## Reviewer Checklist:
